test/react/first-render-only/react-context6.js

```diff
@@ -0,0 +1,47 @@
+var React = require('React');
+// the JSX transform converts to React, so we need to add it back in
+this['React'] = React;
+
+var { Provider, Consumer } = React.createContext(null);
+// this is done otherwise the test fails
+this['_Consumer'] = Consumer;
```

Comment:
This is not a super descriptive comment. *Why* does the test fail? Why do we need a global with this special name here?

Reply:
I'll improve the comment. It's because the getTrials closure doesn't get the right reference unless we put it on the global (because of how we evaluate the getTrials call in `react-test`).

Reply:
I'm mostly confused at where this gets used. I don't see `getTrials` reading something called `_Consumer`.

Reply:
Ah this isn't needed, only for `react-context3.js` where it's used. I'll remove and update the comment on that test.